website/content/docs/api-gateway/consul-api-gateway-install.mdx

```diff
@@ -305,6 +305,49 @@ spec:
 
 </CodeBlockConfig>
 
+If you wish to create a route in a namespace other than the namespace the backendRef is in, you will need to additionally
```

Comment:
```suggestion
If you wish to create a route for a backendRef in a different namespace, you will need to additionally
```
Suggested rephrasing to refocus from the perspective of the route rather than from the backendRef.

I see three possible deployment patterns:
- gateway in separate namespace managed by operator, dev teams manage routes for their own services (requires setting more permissive `allowedRoutes` config on gateway, no reference policy needed for most cases)
- gateway and routes managed by operator in same namespace, pointing to dev team services in other namespaces (requires reference policies)
- routes in a different namespace than both the gateway and services (this is just weird)